docs/quickstart.rst

```diff
@@ -184,18 +219,27 @@ a vastly reduced difference in selection rate:
     >>> constraint = DemographicParity()
     >>> classifier = DecisionTreeClassifier(min_samples_leaf=10, max_depth=4)
     >>> mitigator = ExponentiatedGradient(classifier, constraint)
```

Comment:
objective="balanced_accuracy_score"

Reply:
Sorry, that only works for Postprocessing. I don't believe we have a Moment class for balanced accuracy yet, so we @MiroDudik ?

Should be easy to add though... I'll look into it later today.

Reply:
The `ErrorRate` moment with costs of 0.5 for FP and FN each seems promising. @MiroDudik can you confirm?

Reply:
Hi @romanlutz and @MiroDudik, any updates on this question?

Reply:
Can you confirm that ErrorRate is the right moment to use? When I try to use it with Exponentiated Gradient, I am getting `AttributeError: 'ErrorRate' object has no attribute 'default_objective'`

Reply:
Did you keep the constraint as is and add another arg `objective=ErrorRate(costs={'fp': 0.5, 'fn': 0.5})`? I believe `ErrorRate` wouldn't work as a `constraint` since it doesn't have `default_objective`. 

I suspect you used `ErrorRate` as the `constraint` instead of `DemographicParity`, is that right?

Reply:
> Did you keep the constraint as is and add another arg `objective=ErrorRate(costs={'fp': 0.5, 'fn': 0.5})`? I believe `ErrorRate` wouldn't work as a `constraint` since it doesn't have `default_objective`.
> 
> I suspect you used `ErrorRate` as the `constraint` instead of `DemographicParity`, is that right?

Gotcha! I pushed the update.

Reply:
It still uses DP (ie selection rate parity), and not Equalized Odds (FPR/TPR). As I noted in a different comment earlier that is a problem IMO

Reply:
My apologies, it's been a while since I looked at this file so I walked back too many of the changes. I have changed the constraint back to Equalized Odds.

Reply:
No worries at all! Ah, the numbers look much more interesting now, too 😄 Before, it was all 0s.